pkg/controllers/aws_controller_manager.go

```diff
@@ -29,6 +43,9 @@ func BuildControllerInitializers() map[string]app.ControllerInitFuncConstructor
 
 	controllerInitializers[TaggingControllerKey] = taggingControllerConstructor
 
+	// TODO: remove the following line to enable the route controller
+	delete(controllerInitializers, "route")
```

Comment:
will delete before finishing this PR. I'm keeping this here for now as this does not play well with the [instructed method](https://cloud-provider-aws.sigs.k8s.io/development/) when I'm testing the controller.